README.md

```diff
@@ -10,15 +10,15 @@ Steps to run a local copy of SpartaHackAPI
     ```
       pip3 install virtualenvwrapper
     ```  
-   And add the following three lines to your shell startup file (.bashrc, .profile, etc.)  
+   And add the following three lines to your shell startup file (.zshrc, .bashrc, .profile, etc.)  
```

Comment:
smh...

Reply:
gotta rep that zsh!!

Reply:
If you aren't globbing, you aren't living